apps/src/code-studio/pd/application/teacher1920/Section4ProfessionalLearningProgramRequirements.jsx

```diff
@@ -99,83 +100,104 @@ export default class Section4SummerWorkshop extends LabeledFormComponent {
+            for the costs related to traveling to that location.
+          </p>
+        </div>
+      );
+    } else {
+      return (
         <p>
-          Teachers in this program are required to participate in both:
+          <strong>
+            Your Regional Partner is: {this.state.regionalPartnerName}
+          </strong>
         </p>
-        <ul>
-          <li>
-            One five-day, in-person summer workshop in 2019
-          </li>
-          <li>
-            Up to four one-day, in-person local workshops during the 2019-20 school year
-            (typically held on Saturdays)
-          </li>
-        </ul>
-        {this.radioButtonsWithAdditionalTextFieldsFor('committed', {
-          [TextFields.noExplain]: 'other'
-        })}
-      </div>
-    );
+      );
+    }
   }
 
-  render() {
-    return (
-      <FormGroup>
-        <h3>Section
-          4: {SectionHeaders.section4ProfessionalLearningProgramRequirements}</h3>
-
+  renderAssignedWorkshopList() {
+    if (this.state.partnerWorkshops.length === 0) {
+      return (
         <p>
-          All participants in Code.org’s Professional Learning Program are required to
-          attend a five-day in-person summer workshop. These workshops are hosted locally
-          by Code.org’s Regional Partners. Participants are assigned to workshops based on
-          their Regional Partner. Meals (and in some cases travel costs) will be provided
-          for summer workshops.
+          <strong>
+            Local summer workshop dates have not yet been finalized for your region. Your
+            Regional Partner will be in touch once workshop dates and locations are known.
+          </strong>
         </p>
+      );
+    } else {
+      const options = this.state.partnerWorkshops.map(workshop =>
+        `${workshop.dates} in ${workshop.location}`
+      );
+      options.push(TextFields.notSureExplain);
+      options.push(TextFields.unableToAttend1920);
+      const textFieldMap = {
+        [TextFields.notSureExplain]: 'notSureExplain',
+        [TextFields.unableToAttend1920]: 'unableToAttend'
+      };
 
-        {this.renderContents()}
-      </FormGroup>
-    );
+      return (
+        <div>
+          {
+            this.dynamicCheckBoxesWithAdditionalTextFieldsFor(
+              "ableToAttendMultiple",
+              options,
+              textFieldMap
+            )
+          }
+          {
+            _.intersection([TextFields.notSureExplain, TextFields.unableToAttend1920], this.props.data.ableToAttendMultiple).length > 0 && (
+              <div>
+                {
+                  this.radioButtonsWithAdditionalTextFieldsFor(
+                    "travelToAnotherWorkshop",
+                    {[TextFields.notSureExplain]: 'notSure'},
+                    {
+                      // Ugly hack because this is required but we don't want the asterisk
+                      // to render on a new line
+                      required: false,
```

Comment:
Why would it render on a newline?

What if we inject a `&nbsp;` between the label and the `*` [here](https://github.com/code-dot-org/code-dot-org/blob/e41020607f2c8c485b780720b6b8706a7b0a8b2e/apps/src/code-studio/pd/form_components/FieldGroup.jsx#L59)?